zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -39,12 +38,12 @@
 import java.util.concurrent.RejectedExecutionException;
 import zipkin2.Call;
 import zipkin2.Callback;
+import zipkin2.elasticsearch.ElasticsearchStorage;
 
 import static zipkin2.elasticsearch.internal.JsonReaders.enterPath;
 import static zipkin2.elasticsearch.internal.JsonSerializers.JSON_FACTORY;
 
 public final class HttpCall<V> extends Call.Base<V> {
-  public static final AttributeKey<String> NAME = AttributeKey.valueOf("name");
```

Comment:
are you planning to use the word 'name' in armeria? right now, if we change this here we also need to change in the autoconfig that uses it. This seems unnecessary as we've no current problem with this working.

Reply:
Hmm - I guess in Armeria we'd always use a fully qualified key so it's probably ok if users want to stick to non-qualified ones.

Reply:
cool.. I feel this is a temporary part of code anyway. Mainly I don't want to have to manually regression check everything again, so definitely happy if we can keep this for now